frontend/src/helpers/connector.js

```diff
@@ -5,8 +5,8 @@ import { useState, useEffect } from "react";
 export const MetaMask = new InjectedConnector({ supportedNetworks: [1, 3, 4, 5, 42, 421611] });
 
 const contracts = {
-  1337: { vault: "0x2c15A315610Bfa5248E4CbCbd693320e9D8E03Cc" }, //mainnet-fork
-  3: { vault: "0x3047b2b49f104f38f3b3f1ac9b8df1c62726251e" }, // Ropsten
+  1337: { vault: "0x94f810D5F23F08e772A8d92Dd20f7561d4D6E28b" }, //mainnet-fork
+  3: { vault: "0x6a209bee7225f04b78e8addcd69dda7a7efe5ec0" }, // Ropsten
```

Comment:
this changes shouldnt be here, maybe you need to update with master?

Reply:
Idk why this change is here